### PR TITLE
Update the README.md file

### PR DESCRIPTION
**Pull request by ajessup**
Update the README.md file to mention SPIFFE certificates

### CODE REVIEW

**Comment by ajessup** [comment]
Signed CLA

**Comment by lookuptable** [comment]
Jenkins rebuild auth/presubmit

File: README.md [hunk 1]
```diff
@@ -39,16 +39,16 @@ Service-to-service communication is tunneled through the client side [Envoy](htt
 
 ### Key Management
 Istio Auth provides a per-cluster CA (Certificate Authority) to automate key & cert management. It mainly performs 4 key operations:
-- Generate a key/cert pair for each service account.
-- Distribute key/cert to each pod according to the service account.
+- Generate a [SPIFFE](spiffe.github.io/docs/svid) key/cert pair for each service account.
```
Comment (lookuptable) [inline README.md]: @ajessup can you please also add https scheme here?

**Comment by lookuptable** [comment]
Merging this. Will update the https in a followup PR

**Pull request merged**